scripts/generate-authors.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env fish
+set -e
+
+{
+	cat <<- 'EOH'
+		# This file lists all individuals having contributed content to the repository.
+		# For how it is generated, see `scripts/generate-authors.sh`.
+	EOH
+	echo
+
+    git log --format='%aN <%aE>' | awk '
+    {
+        pos = index($0, "<");
```

Comment:
According to my understanding this logic is to avoid reporting multiple times the same contributor that used different emails. There is an standard way to do this by using a mailmap file (https://blog.developer.atlassian.com/aliasing-authors-in-git/). I don't know if you want to spend time doing that though.